src/main/java/org/jabref/gui/texparser/FileNodeViewModel.java

```diff
@@ -8,7 +8,7 @@
 
 import org.jabref.logic.l10n.Localization;
 
-public class FileNodeViewModel {
+class FileNodeViewModel {
```

Comment:
I would actually still leave this all public, there might be some cases where it could be still needed